drivers/internal/AsyncOp.h

```diff
@@ -1,6 +1,5 @@
 /*
  * Copyright (c) 2018-2019, Arm Limited and affiliates.
- * SPDX-License-Identifier: Apache-2.0
  *
```

Comment:
Why has SPDX line gone ?

Reply:
testing purpose (to see what travis reportS) as stated earlier. I'l lrevert this with rebase